examples/website/i3s-building-scene-layer/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "private": true,
+  "name": "i3s-building-scene-layer-loaders-example",
+  "description": "I3S loaders building scene layer",
+  "version": "0.0.1",
+  "license": "MIT",
+  "scripts": {
+    "start": "vite",
+    "build": "tsc && vite build",
+    "serve": "vite preview"
+  },
+  "dependencies": {
+    "@deck.gl/core": "^8.9.28",
```

Comment:
That is a LOT of dependencies, pretty intimidating for a small example. Are they all used? Do they all need to be declared here (maybe some are already dependencies of dependencies)?

Why do we needs special fonts?

Reply:
fontawesome is used for icons
Other useless dependences have been removed